src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -209,8 +209,8 @@ private void AddLineBreaks(SourceText text, IList<string> lines)
         }
 
         private bool InsertOnEnterTyped(
-            SyntaxTree syntaxTree,
```

Comment:
I would be ok with @sharwell's suggestion.  But then i would want syntaxTree and text to be renamed to make it clear they are not related to "originalDocument".